scripts/etc/init.d/rcS

```diff
@@ -102,10 +102,14 @@ export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bu
 # put PATH in /etc/profile so it's also available when we get a busybox shell
 echo "export PATH=${PATH}" > /etc/profile
 
-mount -t proc none /proc
-mount -t sysfs none /sys
+mount -t proc proc /proc
+mount -t sysfs sysfs /sys
 
-echo /sbin/mdev > /proc/sys/kernel/hotplug
+mount -t tmpfs -o size=64k,mode=0755 tmpfs /dev
+mkdir /dev/pts
+mount -t devpts devpts /dev/pts
+
+echo /opt/busybox/bin/mdev > /proc/sys/kernel/hotplug
```

Comment:
> will try it out, though, but probably tomorrow

:+1: 

I don't know if I've told you before, and otherwise it's now in the public domain, but I use this file for my `online_config` while having that file open in `vim` on my server:
```
echo "Hoi hoi from online-config.txt"
#wait_in_mins=10
#echo "Now we're going to wait for an arbitrary $wait_in_mins minutes to see whether the monitor stays on... "
#for i in `seq $wait_in_mins -1 1`; do
#	sleep 60
#	echo -n "$i.. "
#done
#echo ""
#echo "Let's continue with the installation!"
#echo "mounts:"
#mount
#which mdev
#echo "Let's crash the installer!"
#fail
```
This way I can easily test various things without having to rebuild the installer all the time.